src/Compilers/CSharp/Test/CommandLine/ErrorLoggerTests.cs

```diff
@@ -120,12 +120,12 @@ public class C
           ""locations"": [
           ],
           ""fullMessage"": ""Program does not contain a static 'Main' method suitable for an entry point"",
+          ""isSuppressedInSource"": false,
           ""properties"": {{
             ""severity"": ""Error"",
             ""defaultSeverity"": ""Error"",
```

Comment:
Please see the binskim.exe AnalyzeCommand.cs for its SARIF production, we should keep the compiler and the tool in sync as far as possible (and perhaps have a callable helper that produces SARIF Result instances from a diagnotics.